src/web/highlight.rs

```diff
@@ -6,6 +6,13 @@ use syntect::{
     util::LinesWithEndings,
 };
 
+const CODE_SIZE_LIMIT: usize = 2 * 1024 * 1024;
+const LINE_SIZE_LIMIT: usize = 512;
```

Comment:
512 lines seems a bit short no? But more globally: maybe we should store highlighted files when they are too big?

Reply:
That's 512 characters per line.

Reply:
Ok I see. Let's maybe rename the constant then? Otherwise the name is pretty confusing. Something like `LINE_CHARACTERS_LENGTH_LIMIT`?